roles/capi_cluster/tasks/main.yml

```diff
@@ -1,6 +1,15 @@
 ---
 
 - block:
+    - name: Include kube_prometheus_stack vars
+      include_vars:
+        file: "{{ role_path }}/../kube_prometheus_stack/defaults/main.yml"
+
+    - name: Import prechecks tasks from kube_prometheus_stack
+      import_role:
+        name: kube_prometheus_stack
+        tasks_from: prechecks.yml
```

Comment:
Actually, according to the docs I think it's fine as is: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/include_role_module.html#parameter-tasks_from

Reply:
I think we should actually use `include_role` instead! Fine as is though - you might've been thinking it was an import_task rather than import_role, which would need the treatment you described above...